Green Belt (#48)
AppSecure-nrw/security-belts: green/treatment-of-vulnerabilities-with-severity-high-or-higher.md

> # Treatment of vulnerabilities with severity high or higher

Reviewer: ```suggestion
# Treatment of Vulnerabilities With Severity High or Higher
```

Focus on fixing the most relevant vulnerabilities without being overwhelmed of the potentially great number of low-impact findings.

## Benefits

- Prioritized treatment of most severe vulnerabilities.

> - The software is free of high-impact vulnerabilities, which decreases the chance of denial of service attacks and lost or manipulation of confidential data.

Reviewer: ```suggestion
- The software is free of high-impact vulnerabilities, which decreases the chance of successful denial of service attacks and loss or manipulation of confidential data.
```

## Assessment

> - Show the fix for the latest *n* high criticality findings.

Reviewer: ```suggestion
- Show the fix for the latest *n* high severity findings.
```

- Show how you challenged the efficiency of the implemented fix.

## Related Activities

- [False Positive Treatment](../yellow/false-positive-treatment.md)
- [Test of Containers Regarding Known Vulnerabilities](../green/test-of-container-images-regarding-known-vulnerabilities.md)
- [Test of Components Regarding Known Vulnerabilities](../yellow/test-of-components-regarding-known-vulnerabilities.md)
- [Unit Tests for Security Features](../yellow/unit-tests-for-security-features.md)
- [Test of Client Side Components With Known Vulnerabilities](../orange/test-of-client-side-components-with-known-vulnerabilities.md)
- [Nightly Test of Libraries With Known Vulnerabilities](../orange/nightly-test-of-libraries-with-known-vulnerabilities.md)

> - [Static analysis for all self-written components](static-analysis-for-all-self-written-components.md)

Reviewer: ```suggestion
- [Static Analysis for All Self-Written Components](static-analysis-for-all-self-written-components.md)
```
